ddtrace/opentrace/span.py

```diff
@@ -0,0 +1,41 @@
+from opentracing import Span as OpenTracingSpan
+
+
+class Span(OpenTracingSpan):
+    """Datadog implementation of :class:`opentracing.Span`"""
+
+    __slots__ = []
```

Comment:
Let's use `__slots__` to reduce the footprint / enforce behavior of OpenTracing data structures.